include/nil/crypto3/block/algorithm/decrypt.hpp

```diff
@@ -11,6 +11,7 @@
 
 #include <nil/crypto3/block/cipher_value.hpp>
 #include <nil/crypto3/block/cipher_state.hpp>
+#include <nil/crypto3/block/algorithm/encrypt.hpp>
```

Comment:
Em, what?

Reply:
That's made temporary to use declaration of nop_padding in encrypt.hpp.